### PR TITLE
fixing typos

### PR DESCRIPTION
**Pull request by dret**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,667 +1,667 @@
...
-    <section id="block-activity">
-      <h3>Block Activity</h3>
-      <p>The <code>Block</code> activity is used to indicate that the actor does not want a user (defined in
-	the <code>object</code> property) to be able to interact with objects posted by the user. The
-	server SHOULD prevent the user from interacting with any object posted by the actor.</p>
-    </section>
-  </section>
-  </body>
-</html>
+	<!DOCTYPE html>
```
Comment (dmitrizagidulin) [inline index.html]: Is the re-formatting to use Tabs intentional or accidental?

Comment (dret) [inline index.html]: On 2016-02-02 7:54 , Dmitri Zagidulin wrote:

> Is the re-formatting to use Tabs intentional or accidental?

it's accidental, i am sorry about that.

**Comment by dmitrizagidulin** [comment]
Thanks for fixing the tabs! Looks good.

**Comment by dret** [comment]
recreated in #72

**Pull request closed without merging**